hs_explore/models.py

```diff
@@ -0,0 +1,861 @@
+from django.db import models, transaction
+from hs_core.models import BaseResource
+from django.contrib.auth.models import User, Group
+from hs_core.hydroshare import user_from_id, group_from_id, get_resource_by_shortkey
+
+
+class Status(object):
+    STATUS_NEW = 1
+    STATUS_SHOWN = 2
+    STATUS_EXPLORED = 3
+    STATUS_APPROVED = 4
+    STATUS_DISMISSED = 5
+    STATUS_CHOICES = (
+        (STATUS_NEW, 'New'),
+        (STATUS_SHOWN, 'Shown'),
+        (STATUS_EXPLORED, 'Explored'),
+        (STATUS_APPROVED, 'Approved'),
+        (STATUS_DISMISSED, 'Dismissed')
+    )
+    RECOMMENDATION_LIMIT = 10
+
+
+class KeyValuePair(models.Model):
+    key = models.CharField(max_length=255, editable=False, null=False, blank=False,
```

Comment:
Why aren't we just using a key/value system that exists in DJango? What is the key? what is the value? Names would document this better. This is reimplementing HStore. Need to be explicit about the meanings of fields.